src/legend/svgLegend.ts

```diff
@@ -301,21 +304,25 @@ export class SVGLegend implements ILegend {
             .selectAll(SVGLegend.LegendTitle.selectorName)
             .data(titleData);
 
-        legendTitle.enter()
+        let dataLegendTitle = legendTitle.enter()
             .append("text")
             .classed(SVGLegend.LegendTitle.className, true);
 
-        legendTitle
-            .style({
-                "fill": data.labelColor,
-                "font-size": PixelConverter.fromPoint(data.fontSize),
-                "font-family": SVGLegend.DefaultTitleFontFamily
-            })
+        dataLegendTitle
+            .style(
+                "fill", data.labelColor)
+            .style(
+                "font-size", PixelConverter.fromPoint(data.fontSize))
+            .style(
+                "font-family", SVGLegend.DefaultTitleFontFamily
```

Comment:
Since it's less than 120 symbol we should write this function call into a single line